assets/in

```diff
@@ -147,19 +139,36 @@ prq=$(bitbucket_pullrequest "$repo_host" "$repo_project" "$repo_name" "$prq_id"
 if [ "$prq" != "NO_SUCH_PULL_REQUEST" ] && \
    [ "$prq" != "ALREADY_MERGED" ] && \
    [ "$prq" != "DECLINED" ]; then
-  branch=$(echo "$prq" | jq -r '.fromRef.displayId')
+  from_branch=$(echo "$prq" | jq -r '.fromRef.displayId')
 fi
 
+# calculate source and target commit
+source_commit=$(echo "$prq" | jq -r '.fromRef.latestCommit')
+target_commit=$(echo "$prq" | jq -r '.toRef.latestCommit')
+if [ -z "$source_commit" ]; then
+  log "Unable to determine source commit from merge commit $ref. Please verify depth configuration."
+  exit 1
+fi
+
+if [ -z "$target_commit" ]; then
+  log "Unable to determine target commit from merge commit $ref. Please verify depth configuration."
+  exit 1
+fi
+
+if [[  "$branch" == */from ]]; then
```

Comment:
If the source branch is `/from`, we are merging in the changes from the `$target_commit` so we fake it like `/merge'. This is important for the `get` steps, which is often followed by the pipeline figuring out "what changed" in the PR? Without this step, the PR will miss the changes on the target branch and show them as altered files.